src/client/Utils/index.js

```diff
@@ -48,4 +48,36 @@ export const IndividualRegistrationValidation = (name, emailId, phoneNumber, Sch
     errorObject.isError = true;
   }
   return errorObject;
+}
+
+export const SchoolRegistrationStepOneValidation = (schoolName,emailId,phoneNumber,schoolCoordinatorName,city) => {
+  let errorObject = {   schoolNameError: "", emailIdError:"", phoneNumberError:"", schoolCoordinatorNameError:"", cityError:"", isError: false }
+  //let emailValid = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/.test(emailId);
```

Comment:
why commented?

Reply:
We are not using validations from Utils since we moved validations to same Page, anyway removed commented line